projects/novo-elements/src/elements/time-picker/TimePickerInput.ts

```diff
@@ -80,7 +80,7 @@ export class NovoTimePickerInputElement implements OnInit, ControlValueAccessor
   ngOnInit(): void {
     this.placeholder = this.military ? this.labels.timeFormatPlaceholder24Hour : this.labels.timeFormatPlaceholderAM;
     this.maskOptions = {
-      mask: this.military ? [/\d/, /\d/, ':', /\d/, /\d/] : [/\d/, /\d/, ':', /\d/, /\d/, ' ', /[aApP]/, /[mM]/],
+      mask: this.military ? [/\d/, /\d/, ':', /\d/, /\d/] : [/\d/, /\d/, ':', /\d/, /\d/, ' ', /[aApP上下]/, /[mM午]/],
```

Comment:
Is this the bit that puts AM/PM in front?

Reply:
This controls what the user is allowed to manually type into the input field